app/controllers/chat_channels_controller.rb

```diff
@@ -95,7 +94,7 @@ def block_chat
     authorize chat_channel
     chat_channel.status = "blocked"
     chat_channel.save
-    chat_channel.chat_channel_memberships.map(&:remove_from_index!)
+    chat_channel.chat_channel_memberships.map(&:index_to_elasticsearch)
```

Comment:
There was a lot of linking PRs going on yesterday. Here is the explanation. https://github.com/thepracticaldev/dev.to/pull/6321

> Why not remove them from Elasticsearch? I want to keep all of the memberships in Elasticsearch bc we may want to search for other statuses in the future. It is also easier to stay synced when the general principle is that every membership has a document in Elasticsearch rather than making it a conditional of whether it is or isn't

Reply:
I understand. I was asking if it could be added as a comment as we should try to keep the whys in the code and not in PRs as that knowledge shouldn't be kep there IMO. Probably a better place would be https://docs.dev.to/technical-overview/architecture/ though